build_debian.sh

```diff
@@ -405,6 +405,7 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
 	lsof                    \
 	sysstat                 \
 	xxd                     \
+    wireless-regdb          \
```

Comment:
could you fix this indent?

Reply:
Fixed

Weird that this happened, sorry for the inconvenience